### PR TITLE
Fixed #114

### LINKED ISSUE #114: Redirect to a blank page
Steps to reproduce:
1.Delete all the users present in the database using django admin panel.
2.Now create a superuser with same emailid as that of your github account.
3.Now try to login through your github account ,It will redirect you to a blank page.
p.s:for reproducing this issue you need to make your emailid public,
  
video (https://drive.google.com/file/d/1Jc5FuBF0P6208ioW-kfpBg7E3N0cTZXF/view)
  @AvaisP

### PR DESCRIPTION
**Pull request by vpn1997**
Solved the error of #114 .

### CODE REVIEW

**Comment by PalashTanejaPro** [comment]
Successfully tested!

**Comment by AvaisP** [comment]
Can you explain why this fixes it and why we were having the issue?

**Comment by vpn1997** [comment]
We are having this issue because allauth requires unique email id per user.So if we are having a user with with same emailid as that of our github account in our database( by creating a superuser with same emailid) and then we try to register through our github account,then there will be 2 users with same emailid and causes ambiguity .By setting Unique emailid=False,it changes that setting and now we are allowed to have different users with same emailid .But all these conflicts only happens during debugging phase because github by default needs a unique email id per user.So that fix will only work for debugging phase(where conflict happens). :smile:  @AvaisP

**Comment by AvaisP** [comment]
Sounds good. Will review it within 24 hours

File: django_server/settings.py [hunk 1]
```diff
@@ -167,6 +167,8 @@
 }
 
 SITE_ID = 1
+if(DEBUG):
```
Comment (AvaisP) [inline django_server/settings.py]: Why the parenthesis?

Comment (vpn1997) [reply django_server/settings.py]: just my way of writing code :smile: .I can change if you want to.

Comment (AvaisP) [reply django_server/settings.py]: Yes please change :smile:

**Comment by vpn1997** [comment]
@AvaisP  done :smile:

**Pull request merged**